src/kirby/scss/_global-styles.scss

```diff
@@ -46,12 +46,15 @@
     box-shadow: get-elevation(8) !important;
 }
 
-.kirby-alert-btn {
-    cursor: pointer;
-    --ion-color-primary: #{get-color('black')};
-
-    &.cancel {
-        --ion-color-primary: #{get-color('danger')};
+.kirby-alert{
+    position: fixed;
+    --border-radius: #{$border-radius};
+    --box-shadow: #{get-elevation(8)};
+    
+    .modal-wrapper{
+        max-width: 330px;
```

Comment:
Where does this width come from? Could we extract the value through some variable?

Reply:
It is suppossed to always have this width, but would _variables.scss be a good place? I'm still very new to this codebase.